packages/plugin-char/test/Char.test.ts

```diff
@@ -257,7 +257,7 @@ describe('plugin-char', () => {
                     await testEditor(BasicEditor, {
                         contentBefore: '[]a',
                         stepFunction: (editor: JWEditor) => {
-                            applyFormat(editor, new BoldFormat());
+                            applyFormat(editor, 'bold');
```

Comment:
Well that's sad since I liked the old system better... ^^'

Reply:
But the old system doesn't make it possible for a separate plugin to implement another version of 'bold'...